scripts/cnv_1_genome2gene.py

```diff
@@ -14,61 +14,41 @@ def process_cnv(cpath):
             cnv_cp_only = config_data["cp_only_script"]
             bed_file = config_data["bed_genes"]
             hugo_tsv = config_data["hugo_tsv"]
-            sys.stderr.write("Processing " + cpath + "\n")
+            print("Processing " + cpath, file=sys.stderr)
             root = os.path.basename(cpath)
-            limit = config_data["cnv_min_len"]
-            temp_len = root + ".CNVs_" + str(limit) + "_filtered.bed"
-            len_fh = open(out_dir + temp_len, "w")
-            in_cnv = open(cpath)
-            skip = next(in_cnv)
-            for cnv in in_cnv:
-                ct_dict["total_cnvs"] += 1
-                cnv_data = cnv.rstrip("\n").split("\t")
-                if int(cnv_data[2]) - int(cnv_data[1]) >= limit:
-                    len_fh.write("\t".join(cnv_data[0:5]) + "\n")
-                else:
-                    ct_dict["short_cnvs"] += 1
-                    # sys.stderr.write("Entry too short " + cnv)
-            len_fh.close()
-            temp = root + ".CNVs.Genes"
-            to_genes_cmd = (
-                bedtools
-                + " intersect -a "
-                + out_dir
-                + temp_len
-                + " -b "
-                + bed_file
-                + " -wb > "
-                + out_dir
-                + temp
-            )
+            temp_filtered_fname = root + ".CNVs_0.05_filtered.bed"
+            with open(out_dir + temp_filtered_fname, "w") as temp_filtered:
+                in_cnv = open(cpath)
```

Comment:
Do you ever close this file?

Reply:
I thought when you use `with` syntax, and the end of the block, it gets closed automatically

Reply:
But you never use `with` with this file. With only closes the file that is part of the `with...as...:` line.